rosette/h/BinaryOb.h

```diff
@@ -42,25 +31,25 @@
 
 class BinaryOb : public Ob {
 
-   protected:
-    BinaryOb(int sz, pOb meta, pOb parent):
-        Ob(sz, meta, parent) {}
+    protected:
```

Comment:
Can you make public, protected use an indent of 2 rather than 4 so that they aren't hidden among the things they're annotating?

Reply:
My bad, reformatted with `clang-format` to be consistent. I'm somewhat ambivalent about the indent, but would prefer to maintain it as .clang-format settings.